web/styles/recent_topics.css

```diff
@@ -87,6 +86,7 @@
         }
 
         #recent_topics_filter_buttons {
+            padding-top: 12px;
```

Comment:
Just a small comment -- this commit would definitely benefit from a commit message that briefly explained why moving this padding is useful, so a reviewer doesn't have to guess.

Reply:
Sorry, this to remove it from calculation of `max-height` of `table.`